src/Features/LanguageServer/Protocol/Handler/Completion/CompletionHandler.cs

```diff
@@ -462,10 +462,15 @@ internal CompletionOptions GetCompletionOptions(Document document)
             // 2.  We need to figure out how to provide the text edits along with the completion item or provide them in the resolve request.
             //     https://devdiv.visualstudio.com/DevDiv/_workitems/edit/985860/
             // 3.  LSP client should support completion filters / expanders
+            //
+            // Also don't trigger completion in argument list automatically, since LSP currently has no concept of soft selection.
+            // We want to avoid committing selected item with commit chars like `"` and `)`.
             return _globalOptions.GetCompletionOptions(document.Project.Language) with
             {
                 ShowItemsFromUnimportedNamespaces = false,
-                ExpandedCompletionBehavior = ExpandedCompletionMode.NonExpandedItemsOnly
+                ExpandedCompletionBehavior = ExpandedCompletionMode.NonExpandedItemsOnly,
+                UpdateImportCompletionCacheInBackground = false,
```

Comment:
I have flipped this to false. Since we are not providing unimported items at the moment, there's no point updating the cache at background.